Resources/Maps/saltern.yml

```diff
@@ -6796,6 +6796,11 @@ entities:
   - containers:
       cellslot_cell_container: !type:ContainerSlot {}
     type: ContainerContainer
+  - cellInsertSound: !type:SoundPathSpecifier
+      path: /Audio/Items/pistol_magout.ogg
+    cellRemoveSound: !type:SoundPathSpecifier
+      path: /Audio/Items/pistol_magin.ogg
+    type: PowerCellSlot
```

Comment:
whats up with these changes (and similar?)

Reply:
This is the code's fault, it's not an intentional change.
I got something similar when I tried doing basically the same thing.